### PR TITLE
Add initial PromCon 2016 site.

### PR DESCRIPTION
**Pull request by juliusv**
@fabxc @beorn7 @brian-brazil @grobie 

This is ready for a look, but please don't merge yet without further coordination.

### CODE REVIEW

**Comment by juliusv** [comment]
50% zoom screenshot of entire page:

promcon

**Comment by juliusv** [comment]
@RichiH in case you're interested as well.

File: content/promcon/2016/index.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        <h2>Register to attend</h2>
+        <p>
+          Registration will open soon — fill out the form linked below to get
+          notified when the registration opens.
+        </p>
+        <p>
+          <a class="btn btn-primary btn-lg" href="https://docs.google.com/forms/d/12H6o3vvnnQUJSDV1mQ77EqpU9B64Jk70iFboHwb2y3M/viewform" target="_blank" role="button">Notify Me</a>
+        </p>
+        <p>
+          Admission will likely cost around $50, but we will provide exceptions
```
Comment (brian-brazil) [inline content/promcon/2016/index.html]: We'll probably need this higher, it's better to come down in price than go up so I'd put it nearer €100.

**Comment by beorn7** [comment]
This ignores the existence of PromCon 1. *sniff*

File: content/promcon/2016/index.html (hunk 1 above)
Comment (juliusv) [inline content/promcon/2016/index.html]: I changed it to "around $50 - $100"

**Comment by juliusv** [comment]
@beorn7 :'( I know, but mentioning it here would only serve to confuse people.

**Comment by brian-brazil** [comment]
I'd put it in as the second.

**Comment by juliusv** [comment]
@brian-brazil The second conference? PromCon 1 was pretty unofficial and more of a tiny developer meetup. I don't think it makes sense to put it in the same category of event...

**Comment by RichiH** [comment]
"PromCon16 is happening" can stand without mentioning if it's the first; that would simply leave the question open. If anyone asks, it's always possible to say "it's the second one, but the first one for the public".

**Comment by juliusv** [comment]
I think "first" makes sense in this context, and hopefully helps to generate excitement to join. Anyways, we can always keep it as an inside joke that we were at the "real" first PromCon.

**Comment by RichiH** [comment]
> Anyways, we can always keep it as an inside joke that we were at the "real" first PromCon.

_You_ can, _we_ can't *sniff*

**Comment by grobie** [comment]
What about using promcon.io? Might help in twitter etc. where only the first part of a URL is visible.

**Comment by juliusv** [comment]
@grobie Yeah, I was considering promcon.io or promcon.org as well - though, it would make sense to have a whole separate website then? I could still copy the design of prometheus.io, but without all the other menu items? That would be the fastest way to get there, at least, since we really should start the CFP soon...

Also, what do people think about the suggested closing date for the CFP (April 30). Too short, or ok?

**Comment by RichiH** [comment]
It leaves almost 1.5 months, which sounds fine given the rather limited scope.

**Comment by juliusv** [comment]
Cool, my thinking as well.

**Comment by juliusv** [comment]
Closing in favor of separate site at promcon.io.

**Pull request closed without merging**